lxd/device/usb.go

```diff
@@ -170,21 +170,22 @@ func (d *usb) startVM() (*deviceConfig.RunConfig, error) {
 	runConf := deviceConfig.RunConfig{}
 	runConf.PostHooks = []func() error{d.Register}
 
+	var hostDevicePaths []string
 	for _, usb := range usbs {
-		if !usbIsOurDevice(d.config, &usb) {
-			continue
+		if usbIsOurDevice(d.config, &usb) {
+			devicePath := fmt.Sprintf("/dev/bus/usb/%03d/%03d", usb.BusNum, usb.DevNum)
+			hostDevicePaths = append(hostDevicePaths, devicePath)
```

Comment:
Based on the comment above, I'd expect you'd need to append to `runConf.USBDevice` directly here with a new `USBDeviceItem` struct instance. Then there's no need for `hostDevicePaths`.

Reply:
But you will need to generate a per-actual-device unique `DeviceName` - suggest using `d.name` plus an index ID.

Reply:
Done, have a look please.